src/views/confirmation/confirmation.njk

```diff
@@ -39,6 +39,7 @@
   {% set howIdentityDocsChecked = i18n.option2Confirmation %}
 {% endif %}
 
+<form action="" id="confirmationForm" method="POST">
```

Comment:
You do not need a form tag as there are no input fields for the user to enter data

Reply:
Thanks Troy I've removed that now